client/planning-extension/src/extension.ts

```diff
@@ -269,6 +272,28 @@ const extension: IExtension = {
             },
         };
 
+        // Do not register coverage related CVs as fields, because it overrides current implementation for these fields.
+        // Do not register CVs that aren't considered "custom CVs" also,
+        // because if some are already registered as specific fields they will be overridden.
+        // Definition of a custom CV: client/services/PlanningStoreService.ts:255
+        const customCvs = superdesk.entities.vocabulary
+            .getAll()
+            .toArray()
+            .filter((cv) => !COVERAGE_CVS.includes(cv._id) && !isEmpty(cv.service) && isEmpty(cv.field_type));
```

Comment:
is there a function somewhere `isCustomVocabulary`? If not add it

Reply:
I'd still be useful to have it  `isCustomVocabulary() && !COVERAGE_CVS.includes(cv._id)`. Otherwise reader is left guessing the purpose of cv.service and cv.field_type check